File: iniciante/c/1014-consumo.c
#include <stdio.h>

int main(){
    int distancia_total;
    float combustivel_gasto, consumo_medio;
    scanf("%d %f", &distancia_total, &combustivel_gasto);
    consumo_medio = distancia_total / combustivel_gasto;
    printf("%.3f km/l\n", consumo_medio);
    return 0;
}

File: iniciante/python/1014-consumo.py
def consumo_medio(distancia_total, combustivel_gasto):
    consumo = distancia_total / combustivel_gasto
    return print(f'{consumo:.3f} km/l')


distancia = int(input())
total_combustivel_gasto = float(input())
consumo_medio(distancia, total_combustivel_gasto)
